domains/matrixcoder.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "MatrixCoder0101",
+        "email": "tkgoutamm625@gmail.com"
+    },
+    "record": {
+        "CNAME": "matrixcoder0101.github.io"
```

Comment:
No content is displayed, just default Vercel page.

Reply:
Now I am working on my portfolio

Reply:
It's just a blank NextJS page.

Reply:
> It's just a blank NextJS page.

Sir Please Recheck and Approve my Request.

Reply:
Sir Please Recheck now

Reply:
Looks good.

Reply:
![Screenshot_2023-10-12-08-13-37-77_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/is-a-dev/register/assets/101377369/6fa7ee70-70cf-40fb-b6b5-8a38e365a103)

Reply:
![Screenshot_2023-10-12-12-04-33-82_4d38fce200f96aeac5e860e739312e76.jpg](https://github.com/is-a-dev/register/assets/101377369/71e163df-2c80-4efd-a798-81f233c429f7)

**Can you help me Sir how I apply this domain in GitHub pages 🤔**